src/app/components/shared-components/store/actions/employee-Profile-Details.actions.ts

```diff
@@ -0,0 +1,17 @@
+import { createAction, props } from '@ngrx/store';
+import { EmployeeProfileDetails } from 'src/app/models/hris/EmployeeProfile/employeeProfileDetails.interface';
+
+export const loadEmployeeProfileDetails = createAction(
+  '[Employee Profile] Load Employee Profile Details' ,
+  props<{ employeeId: number }>()
+);
+
+export const loadEmployeeProfileDetailsSuccess = createAction(
+  '[Employee Profile] Load Employee Profile Details Success',
+  props<{ employeeProfileDetails: EmployeeProfileDetails }>()
```

Comment:
Fix: check if you don't need brackets here: EmployeeProfileDetails[] ?

Reply:
it wont be an array